script/deployment/setup/TransferOwnershipToTimelock.s.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.25;
+
+import { OwnableUpgradeable } from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import { Script } from "forge-std/Script.sol";
+import { stdJson } from "forge-std/Script.sol";
+import { EOJsonUtils } from "../../utils/EOJsonUtils.sol";
+
+contract TransferOwnershipToTimelock is Script {
+    using stdJson for string;
+
+    function run() external {
+        vm.startBroadcast();
+        execute();
+        vm.stopBroadcast();
+    }
+
+        // for testing purposes
+    function run(address broadcastFrom) public {
+        vm.startBroadcast(broadcastFrom);
+        execute();
+        vm.stopBroadcast();
+    }
+
+    function execute() public {
```

Comment:
Is this function being executed after creating all/some adapters? 
If so, should we also transfer the ownership on them / the beacon proxy?

Reply:
Yes, it is executed after all scripts. You are right, beacon proxy also needs to be transfered. Updated the script